src/commonMain/kotlin/io/rippledown/model/condition/Is.kt

```diff
@@ -0,0 +1,18 @@
+package io.rippledown.model.condition
+
+import io.rippledown.model.Attribute
+import io.rippledown.model.RDRCase
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Is(val attribute: Attribute, val toFind: String) : Condition() {
+    override fun holds(case: RDRCase): Boolean {
+        val latest = case.getLatest(attribute) ?: return false
+        println("Is: attr text: '${latest.value.text}', toFind: '$toFind'")
+        return latest.value.text == toFind
```

Comment:
what about if x has value 5.1. Would the condition "x is 5.10" evaluate as true?